Src/BootCamp.Chapter/MacFactory.cs

```diff
@@ -1,26 +1,27 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-using BootCamp.Chapter.Computer;
+﻿using BootCamp.Chapter.Computer;
 
 namespace BootCamp.Chapter
 {
     public class MacFactory : DesktopComputer
     {
-        public MacFactory()
-        {
-            Assemble();
-            Console.WriteLine(CompletedAssembly());
-        }
+        Body macBody = new Body();
```

Comment:
This means that all computers from this factory will have the same, shared Body, Ram, Cpu, etc. It looks the same, yes, but it's a DIFFERENT INSTANCE

Reply:
Ok. Now when I moved these down to the assemble function there should be one instance for every computer?